CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+cmake_minimum_required(VERSION 3.3)
+project(EDM4HEP VERSION 0.1.0)
+
+#--- Run the podio class generator and link library -------
+
+find_package(podio REQUIRED HINTS $ENV{PODIO})
+
+execute_process(COMMAND python ${podio_CMAKE_DIR}/../python/podio_class_generator.py edm4hep.yaml edm4hep edm4hep
```

Comment:
Unless `find_package(podio ...)` sets up Python, there should probably be a `find_package(PythonInterp)` before this. Again, maybe Podio should be supplying a CMake function to generate the code from the YAML.